.github/workflows/ci-build.yml

```diff
@@ -38,4 +38,4 @@ jobs:
           curl -sL https://coveralls.io/coveralls-linux.tar.gz | tar -xz \
           && ./coveralls -f ".coverage.xml"
         env:
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+          COVERALLS_REPO_TOKEN: diRoMSg6ajaY373BiIXgXNoD2loQH1r4t
```

Comment:
A better way would be to save the token in the GitHub repo settings. You can create such secrets in GitHub repo Homepage --> settings --> secrets and variables -->  actions --> new repository secret --> put the secret name as COVERALLS_REPO_TOKEN and value as this long token values.

Reply:
Btw, I guess you can somehow wipe this commit to make the token value never appeared in any commit history for security reason.  FYI.

Reply:
Very impressive that you give comment regarding not giving token value directly in the config file which I saw many times during the review of others.  Thanks for your comment here and I will follow your instructions to:
- Invalidate the leaking token
- Setup secret in Github for `COVERALLS_REPO_TOKEN`
- Close current PR and prepare another PR for above changes.